Source/FluentSpreadsheets.GoogleSheets/Extensions/AlignmentExtensions.cs

```diff
@@ -9,7 +11,7 @@ public static string ToGoogleSheetsAlignment(this VerticalAlignment alignment)
             VerticalAlignment.Top => "TOP",
             VerticalAlignment.Center => "MIDDLE",
             VerticalAlignment.Bottom => "BOTTOM",
-            _ => "VERTICAL_ALIGN_UNSPECIFIED"
+            _ => "VERTICAL_ALIGN_UNSPECIFIED",
```

Comment:
Are you sure we need trailing commas?

Reply:
Yeah, other parts of code has them, for me it kinda looks better when they are present.

Reply:
Maybe in enums it's okay, but when it comes to object creation, it just doesn't look good
![image](https://user-images.githubusercontent.com/70411602/186949478-290be8aa-6184-46d8-99b4-15f09dfab9c5.png)

Reply:
I guess it is a matter of getting used to it, looking at this screenshot, I still think that it is better with commas, then without them.

Reply:
They’re just unnecessary, don’t see a point in them